Ollivanders/src/net/pottercraft/Ollivanders2/Prophecy.java

```diff
@@ -86,39 +82,35 @@ public PotionEffect toPotionEffect ()
       lore.add(days + " days, " + hours + " hours, " + minutes + " minutes, and " + seconds + " seconds");
       lore.add("and last for");
       int dur = duration;
-      int durh = (int) (dur / (hour));
+      int durh = (dur / (hour));
       timeTo -= hours * hour;
-      int durm = (int) (dur / (minute));
+      int durm = (dur / (minute));
       timeTo -= minutes * minute;
-      int durs = (int) (dur / (second));
+      int durs = (dur / (second));
       lore.add(durh + " hours, " + durm + " minutes, and " + durs + " seconds");
       return lore;
    }
 
-   public boolean isActive ()
+   boolean isActive ()
    {
       long now = System.currentTimeMillis();
       if (now > this.start && now < this.start + this.duration)
```

Comment:
return (now > this.start && now < this.start + this.duration);

Several methods in this class could have less lines

Reply:
Prophecy is totally broken so I didn't send time cleaning this one up much.